pmd-java/src/main/java/net/sourceforge/pmd/lang/java/symbols/JClassSymbol.java

```diff
@@ -221,6 +181,20 @@ default boolean isClass() {
     }
 
 
+    /**
+     * Returns the toplevel class containing this class. If this is a
+     * toplevel class, returns this.
+     */
+    @NonNull
+    default JClassSymbol getNestRoot() {
```

Comment:
Mabye name this `getToplevel()`? nest root somehow sounds a bit strange (probably because nesting is the directing down and root is the direction up the tree...)